runelite-client/src/main/java/net/runelite/client/plugins/barrows/BarrowsBrotherSlainOverlay.java

```diff
@@ -72,9 +72,11 @@ public Dimension render(Graphics2D graphics)
 			potential.setHidden(true);
 		}
 
+		float BrotherReward = 0;
```

Comment:
nit: please fix the casing `brotherReward`

Reply:
good idea, thanks